### PR TITLE
Adding option to use absolute value in ElementIntegralPostprocessor.

### LINKED ISSUE #28162: Add option to use absolute value for ElementIntegralVariablePostprocessor
### Motivation

Bubbles within a phase-field model can be represented using negative order parameter value. In such cases, the post processor to calculate the bubble area should consider absolute value.

### Design

Add an option to use absolute value in `ElementIntegralVariablePostprocessor`

### Impact

Adding an option to the postprocessor. Should not impact existing tests.

### PR DESCRIPTION
**Pull request by SudiptaBiswas**
Closes #28162

### CODE REVIEW

File: test/tests/postprocessors/element_integral/tests [hunk 1]
```diff
@@ -10,15 +10,23 @@
...
-      detail = 'over a subset of the domain.'
+      detail = 'over a subset of the domain,'
+    []
+
+    [test_abs]
+      type = 'CSVDiff'
+      input = 'element_integral_absolute_test.i'
+      csvdiff = 'out.csv'
+
+      detail = 'and also using the optional abosolute variable value.'
```
Comment (GiudGiud) [inline test/tests/postprocessors/element_integral/tests:29]:
```suggestion
      detail = 'and also using the optional absolute variable value.'
```

**Review by GiudGiud — COMMENTED** [review]
I dont mind this but you can already do it with:
- define a ParsedFunctorMaterial with the absolute value of the variable
- use the element integral pp for functors

File: framework/include/postprocessors/ElementIntegralVariablePostprocessor.h [hunk 2]
```diff
@@ -12,6 +12,9 @@
 #include "ElementIntegralPostprocessor.h"
 #include "MooseVariableInterface.h"
 
+// Forward Declarations
+class ElementIntegralPostprocessor;
```
Comment (GiudGiud) [inline framework/include/postprocessors/ElementIntegralVariablePostprocessor.h:16]:
```suggestion

```
no need we have the header right above

Comment (SudiptaBiswas) [reply framework/include/postprocessors/ElementIntegralVariablePostprocessor.h]: Fixed it

**Comment by SudiptaBiswas** [comment]
> I dont mind this but you can already do it with:
> 
> * define a ParsedFunctorMaterial with the absolute value of the variable
> * use the element integral pp for functors

Agree. But I think this would simplify things. Thanks for reviewing.

**Comment by GiudGiud** [comment]
lets squash #3 in #1 before testing/ merging please

**Comment by SudiptaBiswas** [comment]
> lets squash #3 in #1 before testing/ merging please

Do you mean, only have 1 commit or just squash the third one?

**Comment by SudiptaBiswas** [comment]
> > lets squash #3 in #1 before testing/ merging please
> 
> Do you mean, only have 1 commit or just squash the third one?

I did the former.

**Comment by GiudGiud** [comment]
ok that's fine.
I meant sqaush the 3rd commit into the 1st one

**Review by GiudGiud — APPROVED** [review]
thx for the contribution

**Pull request merged**